yarn.lock

```diff
@@ -2722,6 +2722,7 @@ __metadata:
     "@asteasolutions/zod-to-openapi": "npm:^7.1.0"
     "@cloudflare/workers-types": "npm:^4.20240117.0"
     "@hono/zod-validator": "npm:^0.4.1"
+    "@types/node": "npm:^20.10.4"
```

Comment:
I'm unsure if it's intentional or not, but `@hono/zod-openapi` has it's own `yarn.lock` file which leads to weird build issues where a dependency exists in the root lock file, but not in `@hono/zod-openapi` 🤔 

Unsure if it's simple to remove it or not..

Reply:
This is intended to specify `@hono/zod-validator` on `npm` instead of in on `workspace`.

The current dependencies specify like the following:

https://github.com/honojs/middleware/blob/0f8d2f3d055e7873c22960db6a4a10e20bd75d91/packages/zod-openapi/package.json#L55

But if you do `yarn add @hono/zod-validator` it will be `workspace:^`.

![CleanShot 2025-03-04 at 08 14 33@2x](https://github.com/user-attachments/assets/347041e0-5ff1-4220-b497-6e8ef33da54d)

This will be okay in another general case (?), but it is not suitable for our project. Any good idea?

Reply:
I think `yarn add "@hono/zod-validator@npm:^0.4.1"` should be enough, though it doesn't look like the types from `0.4.3` are compatible with `@hono/zod-openapi`?

So either `@hono/zod-validator@npm:0.4.1"` or `@hono/zod-validator@npm:0.4.2"` might be better

Reply:
Okay! Let's go with `yarn add "@hono/zod-validator@npm:0.4.2"`. It's the latest version which is compatible with `@hono/zod-openapi`.

Reply:
Done 👍🏻